packages/price-exporter/src/cli.js

```diff
@@ -0,0 +1,192 @@
+import fs from 'fs'
+import readline from 'readline'
+import { getCredentials } from '@commercetools/get-credentials'
+import npmlog from 'npmlog'
+import PrettyError from 'pretty-error'
+import yargs from 'yargs'
+
+import PriceExporter from './main'
+import { description } from '../package.json'
+
+process.title = 'price-exporter'
+
+const args = yargs
+  .usage(
+    `
+Usage: $0 [options]
+${description}`,
+  )
```

Comment:
`common-tags` removes the blank line at the beginning, even with `\n` or an actual blank line, so I think it's better to live with this formatting